content/news/051/index.md

```diff
@@ -85,6 +85,27 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 <!-- One-liners for plan items that haven't got their own sections. -->
 
+### RustNL
+
+RustNL happened May 7 & 8 in Delft, Netherlands.
+
+While the conference was not specific to game development,
+organizers contacted rust game development communities
+to showcase games developed in Rust between talks,
+which you can [watch on youtube][rustnl_games_youtube].
+
+Special mention to the following talks, relevant to game development:
+- [The talk from Kevin Boos][rustnl_kevin_boos_talk]: let's build high performance Rust UI
+- [The talk from Ralph Levien][rustnl_xilem_talk] about [Xilem]
+[rustnl_xilem_intro]: a research project on Rust user interface architecture.
```

Comment:
This seems malformed: should this be another bullet point?

Reply:
Ah thanks! it's a misplaced semantic line break